e2e/tests/workspace/workspace-create.spec.ts

```diff
@@ -72,6 +73,26 @@ describe('Creating new workspaces', () => {
     await verifyWorkspaceLinkOnDataPage(newWorkspaceName);
   });
 
+  test('User cannot create a workspace with an old CDR Version without consenting to the restrictions', async () => {
```

Comment:
I'd create a new file for cdr tests like `workspace-cdr.spec.ts` to include this new test and the test below to take advantage of running tests in parallel threads.